hash/rx78_cart.xml

```diff
@@ -109,14 +113,19 @@ The monitor will expect you to load a program from cassette.
 		</part>
 	</software>
 
-	<software name="burger">
-		<description>Burger Shop</description>
+	<software name="cannonbl">
+		<description>Cannon Ball</description>
 		<year>1983</year>
 		<publisher>Bandai</publisher>
-		<info name="usage" value="Z and X to start and move; Space to pick up and put down" />
+		<notes><![CDATA[
+Keyboard controls do not work correctly until you unmap Joystick 2's default keys. (btanb)
+]]></notes>
```

Comment:
The wording really isn’t good if you’re going to leave the comment in.

* Firstly, it isn’t a bug, it’s just that default game control assignments overlap with default keyboard assignments.
* Secondly, you don’t need to clear the assignments for the second controller, just assign it to something that doesn’t conflict with the keyboard.
* Thirdly, as this affects all emulated systems with keyboards, anyone who actually emulates home computers regularly will likely have set this up in general input assignments already and won’t encounter an issue at all.